modules/tensor_mechanics/src/materials/ComputeThermalExpansionEigenStrain.C

```diff
@@ -11,19 +11,19 @@ InputParameters validParams<ComputeThermalExpansionEigenStrain>()
 {
   InputParameters params = validParams<ComputeStressFreeStrainBase>();
   params.addClassDescription("Computes Eigenstrain due to thermal expansion");
-  params.addCoupledVar("Temperature", 273, "Coupled temperature in units of Kelvin");
-  params.addParam<Real>("thermal_expansion_coefficient", 0.0, "Thermal expansion coefficient in 1/K");
+  params.addCoupledVar("temperature", 273, "Coupled temperature in units of Kelvin");
```

Comment:
Are we using `temperature` elsewhere?  I thought we were using `temp` (but maybe it's changed)